newscoop/library/Newscoop/Services/AuditService.php

```diff
@@ -40,14 +40,13 @@ public function __construct(EntityManager $em, UserService $userService)
      */
     public function update(GenericEvent $event)
     {
+        $user = null;
         try {
             $user = isset($event['user']) ? $event['user'] : $this->userService->getCurrentUser();
-            if (!is_int($user)) {
+            if (!is_int($user) && is_object($user) && method_exists($user, 'getId')) {
```

Comment:
why check for `getId` method? user var can be integer or object of type `Newscoop\Entity\User` or something else that you need to check for that method ? `is_object($user)` should be enough imo